aten/src/TH/generic/THTensorMoreMath.cpp

```diff
@@ -28,9 +28,21 @@ int THTensor_(equal)(THTensor *ta, THTensor* tb)
     scalar_t *tbp = tb->data<scalar_t>();
     ptrdiff_t sz = THTensor_(nElement)(ta);
     ptrdiff_t i;
-    for (i=0; i<sz; ++i){
-      if(tap[i] != tbp[i]) return 0;
-    }
+    at::parallel_for(
+        0,
+        sz,
+        TH_OMP_OVERHEAD_THRESHOLD,
+        [&](int64_t begin, int64_t end) {
+          for (auto iter = begin; iter < end; iter++) {
+            if (equal) {
```

Comment:
You wanted
if (!equal)
?

Reply:
Thanks! Sorry for correcting me! Shouldn't write the code at late night...